Sources/MapboxDirections/TollPrice.swift

```diff
@@ -0,0 +1,136 @@
+import Foundation
+import Turf
+
+/**
+ :nodoc:
+ Information about toll payment method.
+ */
+public struct PaymentMethod: Hashable, Equatable {
+    /**
+     Method identifier.
+     */
+    public let identifier: String
+    
+    /**
+     Payment is done by electronic toll collection.
+     */
+    public static let electronicTollCollection = PaymentMethod(identifier: "etc")
+    /**
+     Payment is done by cash.
+     */
+    public static let cash = PaymentMethod(identifier: "cash")
+}
+
+/**
+ :nodoc:
+ Categories by which toll fees are divided.
+ */
+public struct Category: Hashable, Equatable {
+    /**
+     Category name.
+     */
+    public let name: String
+    
+    /**
+     A small sized vehicle.
+     
+     Vehicle size is [standartized](https://en.wikipedia.org/wiki/Expressways_of_Japan#Tolls).
+     */
+    public static let small = Category(name: "small")
+    /**
+     A standard sized vehicle.
+     
+     Vehicle size is [standartized](https://en.wikipedia.org/wiki/Expressways_of_Japan#Tolls).
+     */
+    public static let standard = Category(name: "standard")
+    /**
+     A middle sized vehicle.
+     
+     Vehicle size is [standartized](https://en.wikipedia.org/wiki/Expressways_of_Japan#Tolls).
+     */
+    public static let middle = Category(name: "middle")
+    /**
+     A large sized vehicle.
+     
+     Vehicle size is [standartized](https://en.wikipedia.org/wiki/Expressways_of_Japan#Tolls).
+     */
+    public static let large = Category(name: "large")
+    /**
+     A jumbo sized vehicle.
+     
+     Vehicle size is [standartized](https://en.wikipedia.org/wiki/Expressways_of_Japan#Tolls).
```

Comment:
Typo, and also we should probably mention for now that these categories are for Japan, since “jumbo” doesn’t exist as a category anywhere else.

> In Japan, this is a [standard vehicle size](https://en.wikipedia.org/wiki/Expressways_of_Japan#Tolls).